commerce/domain/pricing/service/classes/PricingServiceSample.cls

```diff
@@ -123,7 +123,6 @@ public class PricingServiceSample extends commercestorepricing.PricingService {
     commercestorepricing.TxnPricingResponseItemCollection txnItemCollection = txnResponse.getTxnPricingResponseItems();
     for (Integer j = 0; j < txnItemCollection.size(); j++) {
       commercestorepricing.TransactionalPricingResponseItem txnItem = txnItemCollection.get(j);
-      txnItem.setLineId(appendField(prefix, txnItem.getLineId()));
       txnItem.setProductId(appendField(prefix, txnItem.getProductId()));
```

Comment:
@adarshjain-sf Since we are not encouraging line id override anymore, can we modify the comment in lines 99 - 102 and remove line id from the list of overridable fields?

Reply:
As discussed, there can be some use cases where overriding the line id is still required. So we are not removing it from the list of overridable fields mentioned in the comment.
But we have modified the comment to tell users that overriding fields containing Salesforce IDs with invalid ids can cause issues.